### ISSUE TITLE
loss is incredibly high

### ISSUE THREAD

**Issue by x64-bit** [x64-bit/ecoli#2]
the loss as of writing this is 120 after 1 epoch.

**that is an immense disappointment.**

it goes down to 40 after 200, but I'm worried it'll overfit. regardless, there's less parameters than there are data points so it should be fine

**Comment by x64-bit** [comment]
fixed, keras is just doing something weird. final RMSE is at most 10, which is the RMSE of a naive approach, second shot I got 4.5
